projects/common/src/utilities/formatters/ordinal/ordinal-formatter.ts

```diff
@@ -0,0 +1,10 @@
+export class OrdinalFormatter {
+  private static readonly ordinals: string[] = ['th', 'st', 'nd', 'rd'];
+
+  public format(value: number): string {
+    const position = value % 10;
+    const ordinals = OrdinalFormatter.ordinals;
+
+    return `${value}${position >= 0 && position < ordinals.length ? ordinals[position] : ordinals[0]}`;
```

Comment:
A switch case would probably make this easier to understand.
0 - probably doesn't need one (although I could see the argument for `th` like this has - certainly for 10th, 20th)
1 - oddball. 1st, 11th, 21st, 31st..
2 - oddball. 2nd, 12th 22nd, 32nd...
3 - oddball. 3rd, 13th, 23rd, 33rd...
I _think_ the rest are all `th` as defined here

Reply:
hah.. let me fix this